microsoft-edge/extensions-chromium/publish/auto-update.md

```diff
@@ -42,7 +42,7 @@ The Microsoft Edge Add-ons website hosts your extensions, and you can update you
 > [!IMPORTANT]
 > Be careful when you update extensions that have a dependency on native binaries.
 
-This article doesn't apply<!-- todo: true? clarify --> to extensions that you publish using the [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) dashboard.  You can use the dashboard to release updated versions to your users and to the Microsoft Edge Add-ons website.  For more information, see [Update a Microsoft Edge extension](../publish/update-extension.md).
+This article doesn't apply<!-- todo: true?  clarify --> to extensions that you publish using the [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) dashboard.  You can use the dashboard to release updated versions to your users and to the Microsoft Edge Add-ons website.  For more information, see [Update a Microsoft Edge extension](../publish/update-extension.md).
```

Comment:
in article "Set an extension to automatically update"
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/auto-update?branch=pr-en-us-3269
it says this article doesn't apply to extensions that are published via Partner Center.  Need discussion of this, does it exist?  Aren't all extensions published via P.C.?  where discussed? in Intro article?  looking...

Reply:
Answer seems to be in
https://learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options - "Extensions that aren't loaded from the Edge Add-ons store are referred to as _externally installed extensions_."  

auto-update.md needs to link there, instead of just linking to Update intro article, which doesn't mention this scenario, but assumes only Partner Center.

Reply:
result:
![image](https://github.com/user-attachments/assets/4c32c3bc-768c-4a1e-8aa2-77a9f8fef343)